Rust/syntax_test_rust.rs

```diff
@@ -779,3 +779,18 @@ macro_rules! alternate_group (
         println!("Test {}!", $a)
     )
 )
+
+macro_rules! kleene_star {
+    ($($arg:tt)+) => (
+        println!($($arg));
```

Comment:
You'll just need to add comments here in like in the lines above that annotate what scopes the character above the `^` has, to ensure the highlighting is working properly.

http://www.sublimetext.com/docs/3/syntax.html#Testing has more info.